perun-core/src/main/java/cz/metacentrum/perun/core/impl/UsersManagerImpl.java

```diff
@@ -978,7 +974,7 @@ public void removeAllUserExtSources(PerunSession sess, User user) {
 
 		// the searchString is already lower cased
 		try {
-			if (Compatibility.isPostgreSql() || Compatibility.isHSQLDB()) {
+			if (Compatibility.isPostgreSql()) {
```

Comment:
Same here, no need to check DB type, since there will be only one.

Reply:
Fixed